backend/tests/v5/unit/utils/helper/strings.test.js

```diff
@@ -93,10 +93,56 @@ const testGenerateHashString = () => {
 	});
 };
 
+const testCapitalizeFirstLetter = () => {
```

Comment:
as mentioned I'm not sure if we need this function, `formatPronouns` seems to do the same thing